packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 -   `BorderControl`: Apply proper metrics and simpler text ([#53998](https://github.com/WordPress/gutenberg/pull/53998)).
+-   `InputControl`: Fix focus style size ([#54394](https://github.com/WordPress/gutenberg/pull/54394)).
```

Comment:
Let's move this entry and the one above under a sub-section, like suggested in https://github.com/WordPress/gutenberg/pull/53998/files#r1324146209

Reply:
Oh yes, I see — thanks!